public/src/components/channelManagement/campaigns/CampaignsForm.tsx

```diff
@@ -46,6 +49,19 @@ const useStyles = makeStyles(({ spacing, typography }: Theme) => ({
   },
 }));
 
+export type CombinedVariant = BannerVariant | EpicVariant | HeaderVariant;
+
+interface TestEnhancements {
```

Comment:
I would just add these fields to the `Test` model in `shared.ts`. Then you can just use `Test` and remove `TestEnhancements`, `CombinedTest`, `EnhancedBannerTest` etc

Reply:
Took a while, but ... done!